portfolio/src/main/webapp/index.html

```diff
@@ -114,8 +114,19 @@ <h2>List of Skills</h2>
 	<section class="sectionRow" id="comments">
 		<h2 class="sectionHeader">Comments</h2>
 		<hr class="round">
-		<ul id="commentsList">
-		</ul>
+		<div id="commentNumberSelector">
+			<label>Number of Comments</label>
+			<select id = "numberOfComments" onchange="changeNumberOfComments()">
+          <option value = "1">one</option>
+          <option value = "2">two</option>
+          <option value = "3">three</option>
+          <option value = "4">All</option>
+        </select>
+		</div>
```

Comment:
Something's not right here with indentation

Reply:
Please look at this on the github review view to see what I'm talking about. This appears to mix tabs and spaces.